cpu/samd21/Makefile.features

```diff
@@ -1 +1,4 @@
+FEATURES_PROVIDED += puf_sram
+CFLAGS += -DHAVE_REBOOT_DETECTION=1
```

Comment:
This would be better located in some CPU header, e.g. periph_cpu.h or cpu_conf.h

Reply:
Well, I did not investigate much but defining in the header does not protect the fallback implementation in puf_sram.c (relates to build order?!)